msg/PositionControllerStatus.msg

```diff
@@ -7,6 +7,4 @@ float32 target_bearing		# Bearing angle from aircraft to current target [rad]
 float32 xtrack_error		# Signed track error [m]
 float32 wp_dist			# Distance to active (next) waypoint [m]
 float32 acceptance_radius	# Current horizontal acceptance radius [m]
-float32 yaw_acceptance		# Yaw acceptance error[rad]
-float32 altitude_acceptance	# Current vertical acceptance error [m]
```

Comment:
> the fields yaw_acceptance and altitude_acceptance that were exclusively used for avoidance and with that also remove some acceptance logic from Navigator

I'd say this has nothing to do with avoidance, we never attempted avoidance on fixed-wing and this is a fixed-wing specific message. But if you don't know what it's useful for then probably no one knows and it doesn't look like it's helping much from the code.

Reply:
All I'm saying is that it was[ only set by the avoidance](https://github.com/PX4/PX4-Autopilot/blob/03b4ad46e8850e929fa0ba6d4d258819ff9a5e34/src/lib/avoidance/ObstacleAvoidance.cpp#L242), so now we can remove it.